s5a/__init__.py

```diff
@@ -25,10 +27,29 @@ def load_ncfile(ncfile):
     :rtype: pandas.core.frame.DataFrame
     """
```

Comment:
Documentation of the new parameter `data_variable_name` is missing.

Reply:
[Done](https://github.com/emissions-api/sentinel5algorithms/pull/25/commits/a5f842cb666b83bf24b2e9515e93b7e2261aa2fd).